quota-notification/src/test/test.iml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
IntelliJ file, let's remove this from the PR.

Reply:
Fixed